app/controllers/api/v1/web_hooks_controller.rb

```diff
@@ -4,18 +4,40 @@ class Api::V1::WebHooksController < ApplicationController
   skip_before_action :verify_authenticity_token
 
   def sendgrid
-    # sendgrid send us an array and this is the way rails does that
+    # Rails automatically handles the JSON data and maps it into the object:  params["_json"]
     events = params['_json']
     events&.each do |event|
       status = event['event']
       message_id = event['uhura_msg_id']
+      # perform_sync_sendgrid_task(message_id, status) # <= Uncomment to test synchronously
```

Comment:
Wouldn't we just run `UpdateSendgridMsgStatusFromWebhookWorker.perform(message_id, status)`? Otherwise we aren't testing the async path when we do that...

Reply:
Correct. I left this in here, commented, b/c it helps _me_ interactively test the flow.  I will remove this comment before the code goes live.